fdbmonitor/local.mk

```
@@ -24,7 +24,7 @@
 fdbmonitor_CFLAGS := -I.
 
 ifeq ($(PLATFORM),linux)
-  fdbmonitor_LDFLAGS := -static-libstdc++ -static-libgcc -lrt
+  fdbmonitor_LDFLAGS := -static-libstdc++ -static-libgcc -lpthread -lrt
```

```suggestion
  fdbmonitor_LDFLAGS := -static-libstdc++ -static-libgcc -pthread -lrt
```

https://stackoverflow.com/questions/23250863/difference-between-pthread-and-lpthread-while-compiling